dev/com.ibm.ws.security.social/src/com/ibm/ws/security/social/tai/OAuthLoginFlow.java

```diff
@@ -45,7 +45,7 @@ public OAuthLoginFlow() {
     }
 
     TAIResult handleOAuthRequest(HttpServletRequest request, HttpServletResponse response, SocialLoginConfig clientConfig) throws WebTrustAssociationFailedException {
-        if (clientConfig instanceof Oauth2LoginConfigImpl && useAccessTokenFromRequest((Oauth2LoginConfigImpl) clientConfig)) {
+        if (clientConfig instanceof Oauth2LoginConfigImpl && SocialUtil.useAccessTokenFromRequest((Oauth2LoginConfigImpl) clientConfig)) {
```

Comment:
The `clientConfig instanceof Oauth2LoginConfigImpl` and the cast to `Oauth2LoginConfigImpl` isn't needed now that the common `useAccessTokenFromRequest()` is being used.